tests/dummy_data/QUANT_RAMP/add_msoa_to_venue.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this file in the most appropriate place? Is it more an utilities script?

And could we include some documentation about what this script is for?

Reply:
Sorry, I've no idea! I suggest we leave it